reactor-tools/src/test/java/reactor/tools/agent/ReactorDebugAgentTest.java

```diff
@@ -136,4 +146,22 @@ public void stackTrace() {
 	private static int getBaseline() {
 		return new Exception().getStackTrace()[1].getLineNumber();
 	}
+
+	static class ProxyMe {
+		static class MyMono extends Mono<Void> {
+			@Override
+			public void subscribe(CoreSubscriber<? super Void> actual) {
+				Mono.<Void>empty().subscribe(actual);
+			}
+		}
+
+		Mono<Void> doSomething() {
+			Mono<Void> myMono = new MyMono();
+			if (myMono == null) {
```

Comment:
is that on purpose? shouldn't it be `if (myMono != null)` to be consistent?

Reply:
it is (see the comment)

Reply:
the comment doesn't help me understand why there's a NPE triggered here

Reply:
This is a dead code that will never get executed

But, since the operators are immutable, I removed the condition and it does not affect the test

Reply:
yeah I know it won't get executed, but since the npe is blatant it becomes a distraction and kind of forces the reader to wonder if the NPE is relevant in the test or not. 

> But, since the operators are immutable, I removed the condition and it does not affect the test

👍